Deliverable 2 Grading

TimeKnow opened the pull request.

serban-petrescu commented:
## Grading

Student: @TimeKnow 

| Aspect                             | Pts | Max |
|------------------------------------|-----|-----|
| Base points                        | 4.0 | 4.0 |
| Domain model                       | 1.5 | 1.5 |
| Conceptual architecture            | 1.5 | 1.5 |
| Package design                     | 1.5 | 1.5 |
| Component and deployment diagram   | 1.0 | 1.5 |
| **Final Grade**                    | 9.5 |     |


Observations:
 - I am not really sure if you will actually need / be able to use both Django templates and React on the client side. Based on your description, you will most likely have a single Django HTML template (but will it actually have any dynamic Django content or will it actually be static), which will be the entry point for the React single page application.  
 - In the deployment diagram, you could add the artifacts that you will have (e.g. a JAR on the server and some HTML + JS on the UI).
 - For completeness, it might be an idea to also depict the database --> Usually it is not located on the same machine as the server.
 - Not sure what you did to the P.R. (it looks like you pushed both to master and to development, then you added and deleted something from development), but it does not show any changes :) I looked on the branch directly.

TimeKnow commented:
Indeed, i don't actually need **React** due to the existence of **Django Template**, but I wanted to study how will they interact (I have seen many industries that use them like this and I was curious). I want to go for **Option 1** from https://www.valentinog.com/blog/tutorial-api-django-rest-react/ .
Regarding the push/delete, I forgot that I should create a new branch and just uploaded directly initialy, added a new branch, deleted them from the main branch and created a pull request
